airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/SnowflakeInsertDestinationAcceptanceTest.java

```diff
@@ -178,6 +196,24 @@ protected void tearDown(final TestDestinationEnv testEnv) throws Exception {
     DataSourceFactory.close(dataSource);
   }
 
+  @Test
+  public void testCheckNoActiveWarehouseConnection() throws Exception {
+    StandardCheckConnectionOutput standardCheckConnectionOutput = runCheck(
+        getConfigNoActiveWarehouseUser());
+
+    assertEquals(Status.FAILED, standardCheckConnectionOutput.getStatus());
+    assertThat(standardCheckConnectionOutput.getMessage()).contains(NO_ACTIVE_WAREHOUSE_ERR_MSG);
+  }
+
+  @Test
+  public void testCheckNoTextSchemaPermissionConnection() throws Exception {
```

Comment:
Unless there's a reason not to let's try to keep the tests the same style since this is how a `check` test is structured in the same file. This also includes not needing to create a separate method for `getConfig` since it's only used once. Instead you can probably add a comment to the test if you need to explain what the `secrets/path_to_file.json` is supposed to represent

https://github.com/airbytehq/airbyte/blob/efa7d08a8a066fead50c747b213d887c38badb38/airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/SnowflakeInsertDestinationAcceptanceTest.java#L193-L196

Reply:
Updated, thanks